src/components/Form/components/FormCheckbox/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { useForm } from 'react-hook-form'
+import { Errors, FormItem, Label, FormCheck } from '../../styled'
+
+type Props = {
+	register: ReturnType<typeof useForm>['register']
+	errors: ReturnType<typeof useForm>['errors']
```

Comment:
What does it mean? I have quite big shortcomings in the types. 🙂

Reply:
The problem began when we divided the form into components - I need to give the form info about what `errors` it needs to handle for `email input field` in `RegisterForm`, but they are defined in `EmailComponent` for universal `Form` component. So I move them to `Props` which allows me to "link" them - `RegisterForm` will correctly evaluate `errors` defined for `email input field` in `EmailComponent`.

Now, Typescript needs them typed, but they are already typed in react-hook-library. Instead of trying to "recreate" or duplicating the original type, I ask it to `ReturnType` from `useForm` function of `react-hook-form` library and I add that I'm looking for specifically ['`errors`'] type.

Reply:
Thanks for the information 👍